moveit_planners/ompl/ompl_interface/CMakeLists.txt

```diff
@@ -27,11 +27,7 @@ target_link_libraries(${MOVEIT_LIB_NAME} ${OMPL_LIBRARIES} ${catkin_LIBRARIES} $
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES COMPILE_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
 
-add_executable(moveit_ompl_planner src/ompl_planner.cpp)
-target_link_libraries(moveit_ompl_planner ${MOVEIT_LIB_NAME})
-set_target_properties(moveit_ompl_planner PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
-
-add_executable(moveit_generate_state_database src/generate_state_database.cpp)
+add_executable(moveit_generate_state_database demos/generate_state_database.cpp)
```

Comment:
i think the word "demos" is overused, isn't this a full feature of OMPL/MoveIt? what makes it a demo?

Reply:
That's fair, I think `demos` was good to distinguish from the library sources. I changed it to `scripts`, which I think is okay, even if it conflicts a bit with python scripts.